src/test/java/io/orkes/conductor/client/e2e/JDBCWorkerTests.java

```diff
@@ -0,0 +1,347 @@
+package io.orkes.conductor.client.e2e;
+
+import com.netflix.conductor.common.metadata.tasks.Task;
+import com.netflix.conductor.common.metadata.tasks.TaskDef;
+import com.netflix.conductor.common.metadata.workflow.StartWorkflowRequest;
+import com.netflix.conductor.common.metadata.workflow.WorkflowDef;
+import com.netflix.conductor.common.metadata.workflow.WorkflowTask;
+import com.netflix.conductor.common.run.SearchResult;
+import com.netflix.conductor.common.run.Workflow;
+import com.netflix.conductor.common.run.WorkflowSummary;
+import io.orkes.conductor.client.ApiClient;
+import io.orkes.conductor.client.MetadataClient;
+import io.orkes.conductor.client.WorkflowClient;
+import io.orkes.conductor.client.http.OrkesMetadataClient;
+import io.orkes.conductor.client.http.OrkesWorkflowClient;
+import io.orkes.conductor.sdk.examples.ApiUtil;
+import org.apache.commons.lang3.RandomStringUtils;
+import org.junit.jupiter.api.*;
+
+import java.util.*;
+import java.util.concurrent.TimeUnit;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import static org.testcontainers.shaded.org.awaitility.Awaitility.await;
+
+public class JDBCWorkerTests {
+    private static WorkflowClient workflowClient;
+    private static MetadataClient metadataClient;
+    private static final String WORKFLOW_NAME = RandomStringUtils.randomAlphanumeric(5);
```

Comment:
We can use static workflowname and task name.

Reply:
Thanks. I'll use taskName also a static variable